packages/x-components/src/design-system/components/button/ghost.scss

```diff
@@ -1,19 +1,19 @@
 .x-button--ghost.x-button,
 .x-button--ghost .x-button {
   // color
-  --x-color-background-button: var(--x-color-background-button-ghost);
-  --x-color-border-button: var(--x-color-border-button-ghost);
-  --x-color-text-button: var(--x-color-text-button-ghost);
+  --x-color-background-button-default: var(--x-color-background-button-ghost);
+  --x-color-border-button-default: var(--x-color-border-button-ghost);
+  --x-color-text-button-default: var(--x-color-text-button-ghost);
 }
 
 .x-button--ghost.x-button--ghost-start.x-button,
 .x-button--ghost.x-button--ghost-start .x-button {
-  // spacing
-  padding-inline-start: 0;
+  // size
+  --x-size-padding-left-button-default: 0;
```

Comment:
I think this change and the other one below will "harm" the out-of-the-box `rtl` support.

Reply:
I changed the properties of `padding`, `border-width` and `margin`.
I left the names of the tokens to keep them more understandable, as this doesn't affect the CSS.